docs/list.json

```diff
@@ -22,6 +22,7 @@
 				"How to dispose of objects": "manual/en/introduction/How-to-dispose-of-objects",
 				"How to create VR content": "manual/en/introduction/How-to-create-VR-content",
 				"How to use post-processing": "manual/en/introduction/How-to-use-post-processing",
+				"How to use node material": "manual/en/introduction/How-to-use-node-material",
```

Comment:
I would suggest to capitalize both words, i.e. write `How to use Node Material`.